examples/object_subclass/author.rs

```diff
@@ -20,11 +19,17 @@ mod imp {
         surname: RefCell<String>,
     }
 
+    #[glib::signals(wrapper_type = super::Author)]
+    impl Author {
+
+        #[signal]
+        fn awarded(&self) {}
+    }
+
     #[glib::derived_properties]
     impl ObjectImpl for Author {
         fn signals() -> &'static [Signal] {
-            static SIGNALS: OnceLock<Vec<Signal>> = OnceLock::new();
-            SIGNALS.get_or_init(|| vec![Signal::builder("awarded").build()])
+            Self::derived_signals()
```

Comment:
It would be nice to add a derived_signals macro as well, not much work and would make the migration to use the future signals macro more appealing